modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -8,39 +8,197 @@
 
 Configure disk partitioning for a managed cluster using a `SiteConfig` CR and {ztp-first}. The  disk partition details in the `SiteConfig` CR must match the underlying disk.
 
-[NOTE]
+[IMPORTANT]
 ====
-Use persistent naming for devices to avoid device names such as `/dev/sda` and `/dev/sdb` being switched at every reboot. You can use `rootDeviceHints` to choose the bootable device and then use same device for further partitioning.
+You must complete this procedure at installation time.
 ====
 
 .Prerequisites
 
-* You have installed the OpenShift CLI (`oc`).
+* Install Butane.
+
+.Procedure
 
-* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+. Create the `storage.bu` file.
```

Comment:
Would recommend that you add something like: "...by using the following example YAML file:"